tests/snuba/api/endpoints/test_organization_events_histogram.py

```diff
@@ -127,7 +128,7 @@ def test_bad_params_missing_fields(self):
         response = self.do_request(query)
         assert response.status_code == 400
         assert response.data == {
-            "field": ["This list may not be empty."],
+            "field": [ErrorDetail(string="This field is required.", code="required")],
```

Comment:
@Zylphrex I saw that you wrote this code originally, do you know if the frontend is relying on this specific value?

Reply:
Can confirm that the frontend does not depend on the specific value.